js/App.js

```diff
@@ -1,13 +1,10 @@
 import React, { Component } from "react";
 import { StyleSheet, Text, View } from "react-native";
```

Comment:
you are not using any of these components coming from react native. Please remove them.